addons/project/static/tests/tours/project_update_tour_tests.js

```diff
@@ -72,91 +72,9 @@ tour.register('project_update_tour', {
     trigger: ".o_kanban_record .oe_kanban_content",
     extra_trigger: '.o_kanban_project_tasks',
     run: "drag_and_drop .o_kanban_group:eq(1) ",
-}, {
-    trigger: '.o_back_button',
-    content: 'Go back to edit the project created',
-}, {
-    // an invisible element cannot be used as a trigger so this small hack is mandatory for the next step
-    trigger: '.o_kanban_record:contains("New Project")',
-    run: function () {
-        $('.o_kanban_record:contains("New Project") .o_dropdown_kanban').css('visibility', 'visible');
-    }
-}, {
-    trigger: '.oe_kanban_global_click :contains("New Project") .o_dropdown_kanban',
-    content: 'Open the project dropdown'
-}, {
-    trigger: '.o_kanban_record:contains("New Project") .dropdown-menu a:contains("See More")',
-    content: 'Start editing the project',
-}, {
-    trigger: '.nav-item:contains("Invoicing") .nav-link',
-    content: 'Start the assignement of a default sale order id to the project'
-}, {
-    trigger: '.o_form_button_edit',
-    content: 'Start editing the project'
-}, {
-    trigger: 'div[name="sale_line_id"]',
-    content: 'Select the first sale order of the list',
-    run: function (actions) {
-        actions.text('Prepaid', this.$anchor.find('input'));
-    },
-}, {
-    trigger: 'ul.ui-autocomplete > li:first-child > a',
-    content: 'Select the first item on the autocomplete dropdown'
-}, {
-    trigger: '.o_form_button_save',
-    content: 'Save the modifications'
-}, { 
-    trigger: '.o_back_button',
-    content: 'Go back to the kanban view the project created',
-}, {
-    trigger: '.oe_kanban_global_click :contains("New Project")',
-    content: 'Open the project'
```

Comment:
@cedricpri I'd have thought this could be moved to the sale_timesheet_tour, isn't it possible ?